src/openforms/prefill/contrib/haalcentraal/plugin.py

```diff
@@ -72,25 +72,35 @@ def _get_values_for_bsn(
 
         return values
 
-    @classmethod
-    def get_prefill_values(
-        cls, submission: Submission, attributes: Sequence[str]
-    ) -> dict[str, Any]:
+    def get_identifier_value(
+        self, submission: Submission, identifier_role: str
```

Comment:
I think that `identifier_role` should be a strict type hint: `Literal["main", "authorised_person"]`